client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -143,7 +131,11 @@ const getInventoryAdjustmentReasonInputColumn = (
           value={value}
           width={column.width}
           onChange={onChange}
-          adjustment={getAdjustment()}
+          adjustmentType={
+            rowData.snapshotNumberOfPacks > (rowData?.countedNumberOfPacks ?? 0)
+              ? AdjustmentTypeInput.Reduction
+              : AdjustmentTypeInput.Addition
+          }
```

Comment:
Since we're don't have a "None" option anymore, will default to "Addition" except in that one specific case, so no longer need a separate method.